docs/shared/getting-started/nx-setup.md

```diff
@@ -22,41 +26,49 @@ npx create-nx-workspace --preset=core
 npx create-nx-workspace --preset=ts
 ```
 
-Some presets set up applications, e2e tests, etc.
+Some presets are set up to build apps with common frameworks
 
 ```bash
+npx create-nx-workspace --preset=@nrwl/remix
+
 npx create-nx-workspace --preset=react
+
 npx create-nx-workspace --preset=react-native
+
 npx create-nx-workspace --preset=angular
 ```
```

Comment:
this section feels weird/not relevant the page focus is getting started with first-time nx workspace
are we showing people can use the --preset flag
are we recommending this usage

maybe this should best be moved into a "preset" page somewhere?